docs/user/videos-and-tutorials/guides/migrating-from-avado.md

```diff
@@ -0,0 +1,251 @@
+# Transitioning To Dappnode
+
+:::caution Warning
+**Important:** Before following this guide to install DappnodeOS on your Avado device, make sure to first backup any data if you are running a validator or any other dApp.
+**The migration process will otherwise wipe all existing data.**
+
+:::
+
+## Introduction
+
+This guide was adapted from the original [HOPR Migration Guide](https://github.com/hoprnet/hopr-docs/blob/d8d1ec05e5938eb29254188b711eec47a71a92ab/versioned_docs/version-v2.1.3/node/using-avado.md) & will walk you through the process of migrating your Avado node to a Dappnode.
```

Comment:
Hey @Hydepwns! What do you think about adding the text in bold here? 

This guide was adapted from the original HOPR Migration Guide & will walk you through the process of migrating your Avado node to a Dappnode **, due to the discontinuation of support for HOPR from AVADO**

I can merge it after this is added, or add it myself later if you prefer.

Reply:
adding this now sir

Reply:
Semantic update to:
"... & will walk you through the process of migrating your Avado hardware to DappnodeOS, due to the discontinuation of support for HOPR from AVADO."